worker/src/durable.rs

```diff
@@ -708,9 +708,9 @@ impl ScheduledTime {
 }
 
 impl From<i64> for ScheduledTime {
-    fn from(offset: i64) -> Self {
+    fn from(in_millis: i64) -> Self {
```

Comment:
This might actually be slightly better worded:

```suggestion
    fn from(offset_ms: i64) -> Self {
```